editor/src/com/talosvfx/talos/editor/addons/scene/apps/routines/RoutineStage.java

```diff
@@ -45,14 +46,19 @@ public class RoutineStage extends DynamicNodeStage<RoutineStageData> implements
     private boolean paused = false;
     @Getter
     private boolean playing;
-    private ScenePreviewApp scenePreviewApp;
+
+    @Getter
+    private ScenePreviewApp currentScenePreviewApp;
```

Comment:
These should not be stored anywhere. The lifecycle of any app is independent from any other app. If we close a scene preview app, this is now going to be stale in this map.